### ISSUE TITLE
Cannot access EpiRSV or EpiPox

### ISSUE THREAD

**Issue by bernice-chan-ukhsa** [Wytamma/GISAIDR#43]
I cannot access EpiRSV nor EpiPox with login(), with error returning as :

```
Sending request:
 Method -> GET
 URL -> https://www.epicov.org/epi3/frontend?
 data -> 
Extracting 'name="sid" value='([^']*)' from '
        <html>

        <body'
Sending request:
 Method -> GET
 URL -> https://www.epicov.org/epi3/frontend?sid=A6D84BBV92RZAPGK7KRM732YHQ3E2GDO
 data -> 
Extracting '"WID"] = "([^"]*)' from '<!DOCTYPE html PUBLIC "-//W3C/'
Extracting 'PID"] = "([^"]*)' from '<!DOCTYPE html PUBLIC "-//W3C/'
Extracting 'sys.getC\('(.*)').call\('doLogin'' from '<!DOCTYPE html PUBLIC "-//W3C/'
Sending request:
 Method -> POST
 URL -> https://www.epicov.org/epi3/frontend?
 data -> sid=A6D84BBV92RZAPGK7KRM732YHQ3E2GDO&wid=wid_rtopk6_70wl&pid=pid_rtopk6_70wm&data=%7B%22queue%22%3A%5B%7B%22wid%22%3A%22wid_rtopk6_70wl%22%2C%22pid%22%3A%22pid_rtopk6_70wm%22%2C%22cid%22%3A%22c_rtopk6_64%22%2C%22cmd%22%3A%22doLogin%22%2C%22params%22%3A%7B%22login%22%3A%22bernice.chan%22%2C%22hash%22%3A%2207b6aad196a9663e7e5f9b7adeae37f8%22%7D%2C%22equiv%22%3Anull%7D%5D%7D&ts=1682602237000&mode=ajax
Extracting '\('(.*)')' from 'sys.goPage('pid_rtopk6_70wn')'
Sending request:
 Method -> GET
 URL -> https://www.epicov.org/epi3/frontend?sid=A6D84BBV92RZAPGK7KRM732YHQ3E2GDO&pid=pid_rtopk6_70wn
 data -> 
Extracting 'sys.call\('(.{5,20})','Go'' from '<!DOCTYPE html PUBLIC "-//W3C/'
Sending request:
 Method -> GET
 URL -> https://www.epicov.org/epi3/frontend?sid=A6D84BBV92RZAPGK7KRM732YHQ3E2GDO&wid=wid_rtopk6_70wl&pid=pid_rtopk6_70wm&data=%7B%22queue%22%3A%5B%7B%22wid%22%3A%22wid_rtopk6_70wl%22%2C%22pid%22%3A%22pid_rtopk6_70wn%22%2C%22cid%22%3A%22c_rtopk6_wv%22%2C%22cmd%22%3A%22Go%22%2C%22params%22%3A%7B%22page%22%3A%22rsv%22%7D%2C%22equiv%22%3Anull%7D%5D%7D&ts=1682602238000&mode=ajax
 data -> 
Extracting '\('(.*)')' from 'sys.goPage('pid_rtopk6_70ya')'
Sending request:
 Method -> GET
 URL -> https://www.epicov.org/epi3/frontend?sid=A6D84BBV92RZAPGK7KRM732YHQ3E2GDO&pid=pid_rtopk6_70ya
 data -> 
Extracting 'sys-actionbar-action.*" onclick="sys.getC\('([^']*)' from '<!DOCTYPE html PUBLIC "-//W3C/'
Sending request:
 Method -> GET
 URL -> https://www.epicov.org/epi3/frontend?sid=A6D84BBV92RZAPGK7KRM732YHQ3E2GDO&wid=wid_rtopk6_70wl&pid=pid_rtopk6_70ya&data=%7B%22queue%22%3A%5B%7B%22wid%22%3A%22wid_rtopk6_70wl%22%2C%22pid%22%3A%22pid_rtopk6_70ya%22%2C%22cid%22%3A%22c_rtopk6_f5w%22%2C%22cmd%22%3A%22Go%22%2C%22params%22%3A%7B%22link%22%3A%22page_rsv.RSVBrowsePage%22%7D%2C%22equiv%22%3Anull%7D%5D%7D&ts=1682602239000&mode=ajax
 data -> 
Extracting '\('(.*)')' from 'sys.goPage('pid_rtopk6_70yc')'
Sending request:
 Method -> GET
 URL -> https://www.epicov.org/epi3/frontend?sid=A6D84BBV92RZAPGK7KRM732YHQ3E2GDO&pid=pid_rtopk6_70yc
 data -> 
Extracting 'div class="sys-datatable" id="(.*)_table' from '<!DOCTYPE html PUBLIC "-//W3C/'
Extracting 'sys.createComponent\('(.{5,20})','RSVSearchComponent'' from '<!DOCTYPE html PUBLIC "-//W3C/'
Extracting '.createFI\('(.*)','.*Widget','covv_location' from '<!DOCTYPE html PUBLIC "-//W3C/'
Extracting '.createFI\('(.*)','.*Widget','mutation' from '<!DOCTYPE html PUBLIC "-//W3C/'
Extracting '.createFI\('(.*)','.*Widget','nuc_mutation' from '<!DOCTYPE html PUBLIC "-//W3C/'
Error in ceid[[1]] : subscript out of bounds
```

I can access the EpiCoV database though with the same credentials.

I am running R studio 4.2.3 and GISAIDR 0.9.6.

**Comment by Wytamma** [comment]
Hi @bernice-chan-ukhsa!

Thanks for raising the issue. These databases are untested and so not as reliable as EpiCoV. From the error about I can see that a recently added feature is breaking the access. 

I will fix this and get back to you asap

**Comment by Wytamma** [comment]
Solved in b6b8443 (v0.9.7)
